rpc/tests/wasm.rs

```diff
@@ -0,0 +1,25 @@
+#![cfg(target_arch = "wasm32")]
+
+use celestia_rpc::client::Client;
+use celestia_rpc::prelude::*;
+use wasm_bindgen_test::*;
+
+const CELESTIA_RPC_URL: &str = "ws://localhost:26658";
+
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
+
+#[wasm_bindgen_test]
```

Comment:
We haven't enabled wasm tests for RPC as there wasn't anything to test yet, so these won't run in CI.
Please add a similar `wasm-pack` line with rpc crate to our [`workflows/ci.yaml`](https://github.com/eigerco/lumina/blob/main/.github/workflows/ci.yml#L98).

Also I see you're setting `run_in_browser`. Is this necessary? At glance these could run in node.js and that I think is slightly less heavy than entire browser.

Reply:
run_in_browser is unfortunately necessary, because jsonrpsee uses gloo for the networking which uses browser WebSocket API